### PR TITLE
Cleanup sandcastle and docs for ground atmosphere

### PR DESCRIPTION
**Pull request by OmarShehata**
### What I did

I think the current Sandcastle names of "Atmosphere" and "Atmosphere color" are confusing, because it implies they're different aspects of the same thing. But "Atmosphere color" only changes the color of the sky as seen from the ground or the atmosphere around the ellipsoid. Whereas "Atmosphere" is specifically about the ground atmosphere. 

I renamed them to "Ground Atmosphere" and "Sky Atmosphere" respectively. Both will show up when a user searches for "Atmosphere" so it will be obvious there are two different ways of affecting the atmosphere. 

I also updated some things that were copied from the Sky Atmosphere example to the Ground one, and added links to the Sandcastle in the docs. 

### One problem left

I updated the `showGroundAtmosphere` doc to say that it's only visible when the camera is at a distance between `lightingFadeOutDistance` and `lightingFadeInDistance` but the doc for those say:

> The distance where everything becomes lit. This only takes effect when `enableLighting` is `true`.

Which seems to contradict this, since you can see it does affect the ground atmosphere whether or not lighting is enabled:

https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/?src=Atmosphere.html

So I'm not sure whether this is expected behavior that `lightingFadeInDistance` controls both lighting and ground atmosphere and if the doc just needs to be updated, or if there's supposed to be a separate control for ground atmosphere?

@bagnell @lilleyse

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @OmarShehata!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/ground-atmosphere-cleanup/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/Globe.js [hunk 1]
```diff
@@ -136,7 +136,9 @@ define([
         this.enableLighting = false;
 
         /**
-         * Enable the ground atmosphere.
+         * Enable the ground atmosphere, which is drawn over the globe when viewed from a distance greater than <code>lightingFadeOutDistance</code>. 
+         *
+         * @demo {@link https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Ground%20Atmosphere.html|Ground atmosphere demo on Sandcastle}
```
Comment (lilleyse) [inline Source/Scene/Globe.js]: `on Sandcastle` to `in Sancastle`?

**Comment by lilleyse** [comment]
> So I'm not sure whether this is expected behavior that lightingFadeInDistance controls both lighting and ground atmosphere and if the doc just needs to be updated, or if there's supposed to be a separate control for ground atmosphere?

I think it's expected that it controls both, and the doc just needs to be updated to say it takes effect when `enableLighting` or `showGroundAtmosphere` are true.

File: CHANGES.md [hunk 2]
```diff
@@ -3,6 +3,9 @@ Change Log
 
 ### 1.51 - 2018-11-01
 
+##### Breaking Changes :mega:
+* Renamed Atmosphere Sandcastle example to [Ground Atmosphere](https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Ground%20Atmosphere.html) and Atmosphere color Sandcastle example to [Sky Atmosphere](https://cesiumjs.org/Cesium/Apps/Sandcastle/index.html?src=Sky%20Atmosphere.html).
```
Comment (mramato) [inline CHANGES.md]: Sandcastle examples do not qualify as breaking changes, they are completely outside of the API.

Comment (lilleyse) [reply CHANGES.md]: Hmm that's true... @OmarShehata could you remove the similar breaking change note from `1.50`?

**Comment by OmarShehata** [comment]
@lilleyse I removed the Sandcastle renames from breaking changes, both for this and for the 1.50 release from `CHANGES.md`.

**Comment by OmarShehata** [comment]
@lilleyse this should be good to merge.

**Comment by lilleyse** [comment]
Ah thanks for the reminder!

**Pull request merged**